introduction.md

```diff
@@ -0,0 +1 @@
+Thomas learning to use github on codeschool.  Mostly for fun not for work.  Happy thankgiving.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.